lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -266,6 +266,12 @@ def install_puppet_on(hosts, opts={})
               if host['platform'] =~ /el-(5|6|7)/
                 relver = $1
                 install_puppet_from_rpm_on(host, opts.merge(:release => relver, :family => 'el'))
+              elsif host['platform'] =~ /sles-(11|12)/
+                relver = $1
+                # Need to get rid of whatever Puppet might have shipped on the
+                # upstream images
+                host.uninstall_package('puppet')
```

Comment:
I'm down with that. @johnduarte do you know if there will be any issues running this command on our internal SLES platforms when puppet isn't installed?